### ISSUE TITLE
Alt Text in images

### ISSUE THREAD

**Issue by JayVii** [toddsundsted/ktistec#52]
In the context of accessibility, it is nice to supplement images you post with ALT-texts. For mastodon and many other fediverse servers, there is typically a specific field in the posting work flow for that.

Image attachments work slightly different in ktistec, where you can add captions to an image, however it is not the same as the ALT parameter in HTML or the ALT functionality in mastodon.

ALT text from e.g. mastodon posts are not transfered to ktistec. And Ktistec's captions are not translated to ALT text in other fediverse servers, but show up as normal text in the post.

I realize that captions and ALT text serve two completely different purposes. Captions give a bit of context, while ALT text describe the images for those that cannot see them (or those that choose to block images). So it might not make sense to use captions for ALT text anyways, but should be two different features.

Not entirely sure how to tackle this best, but the HTML param would make sense on the ktistec side. I don't know how to transfer these via activity pub.

**Comment by toddsundsted** [comment]
support for displaying alt-text fixed in 05fdba58. support for adding alt-text isn't yet planned. it wouldn't be hard to copy the figure caption to the `alt` attribute, but that doesn't seem to be the best practice. we probably need some way to edit it independently of the figure caption, but the editor doesn't support that.

**Comment by toddsundsted** [comment]
support for adding/editing alt text in c0e350bb. it took a while to figure out how to do this.

**Comment by JayVii** [comment]
Thanks a lot for this feature!
